src/versions.js

```diff
@@ -16,13 +31,6 @@ const versions = {
                     designLibraryDoc: 'https://www.sketch.com/s/8016dbb8-1bb5-4fac-bea4-09f39aca414a',
                 },
```

Comment:
Should we add `switch` as well?

Reply:
Fixed.

Reply:
@groberts314, I change this file pretty simnifically. It's my belief now, that it provides a little more information a little more accurately.

Reply:
Sounds good @cbrewer1689 .  I see switch is in here now, so I'm gonna resolve this thread.